strictEquals.js

```diff
@@ -1,10 +1,24 @@
-function strictEquals(num1, num2) {
-  let num1Type = typeof num1
-  let num2Type = typeof num2
+// ***Solution 1***
+// function strictEquals(num1, num2) {
+//   let num1Type = typeof num1
+//   let num2Type = typeof num2
 
-  if (num1 == num2 && num1Type == num2Type) {
+//   if (num1 == num2 && num1Type == num2Type) {
+//     return true
+//   } 
+//   else {
+//     return false
+//   } 
+// }
```

Comment:
commented code not really needed :(

and this code was your correct code. as we are not allowed to use !== or === in the answer.